src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -22,12 +22,12 @@ public FixturesLoader(String path, Root config) {
     public Map<String, Fixture> load() {
         try {
             return Files
-                .walk(Paths.get(path))
```

Comment:
Let's revert the cosmetic changes - we need to push these things as a separate PRs